pkg/graph/resolvers/nda_agreement_test.go

```diff
@@ -0,0 +1,41 @@
+package resolvers
```

Comment:
Can you add a test case to this file to test the case of agreeing to an already agreed-to NDA, and ensuring it doesn't update it? (Looks like it shouldn't update the `agreed_dts`, and maybe it shouldn't even update `modified_dts`?)

Reply:
@ClayBenson94, sounds good, I will add the test.

I'm inclined to not handle not updating the `modified_dts` to be in line with the rest of the app. (When we apply changes, we don't do anything to check if the supplied values are equivalent to what's already in the database, we just take the input and attempt to update, capturing the users intent to update). I think the fact that agreed_dts won't be updated in this case should be sufficient. Thoughts?

Reply:
I think that's fine. Also in-line with the rest of the app, but a test case around not updating the `agreed_dts` still sounds valuable!

Reply:
I added permutations of the use cases to Unit tests.